### PR TITLE
make product card

### PR DESCRIPTION
**Pull request by maks-golovatyuk**
https://maks-golovatyuk.github.io/layout_product-cards/
https://maks-golovatyuk.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--color_yellow :nth-child(-n + 4) {
```
Comment (witflash) [inline src/style.css:65]: You should complete this requirement:
image

So you should have all possible modifiers: `.stars--1`, `.stars--2`, ..., `.stars-5`.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  color: #616070;
+}
+
+.card__number-content {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
```
Comment (witflash) [inline src/style.css:104]: If you use the color more than one time, move it to the CSS variable and use the variable name.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+.card__link {
+  padding: 10px 60px;
+  text-decoration: none;
+  color: #fff;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```
Comment (witflash) [inline src/style.css:125]: The same for this color.

**Review by witflash — CHANGES_REQUESTED** [review]
Pay attention to my comments, please.

File: src/style.css (hunk 3 above)
Comment (maks-golovatyuk) [reply src/style.css]: All corrected

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,60 @@
...
+        <img
+          class="card__imac-img"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+      </div>
+
+      <div class="card__content">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
```
Comment (polosanya) [inline src/index.html:22]: image

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  margin-bottom: 16px;
+}
+
+.card__feedback {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
```
Comment (polosanya) [inline src/style.css:54]: Move these styles to a separate file 
image

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 :nth-child(-n + 4) {
```
Comment (polosanya) [inline src/style.css:67]: What about other modifiers?

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+  background-image: url(images/star-active.svg);
+
+}
+
+.stars :last-child {
+  margin-right: 0;
+}
+
+.card__review {
+  font-family: "Roboto", sans-serif;
```
Comment (polosanya) [inline src/style.css:77]: You can just inherit it. No need to duplicate this property everywhere

**Review by polosanya — CHANGES_REQUESTED** [review]
Button font is a bit different. Check yourself with the design
image
And pay attention to other comments to improve your code)

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,60 @@
...
+        <p class="card__review">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__text-content">
+          Price
+        </p>
+        <p class="card__number-content">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:46]:
```suggestion
        </p>
        
        <p class="card__number-content">
```

**Review by Anastasiia-Svintsova — APPROVED** [review]

File: src/style.css (hunk 7 above)
Comment (maks-golovatyuk) [reply src/style.css]: All corrected, plas check

**Review by polosanya — APPROVED** [review]
Good job!

**Review by nazarmatsevych — APPROVED** [review]

File: src/style.css (hunk 3 above)
Comment (maks-golovatyuk) [reply src/style.css]: Please check. All is done

**Review by witflash — APPROVED** [review]

**Pull request closed without merging**